exporter/awsemfexporter/config.go

```diff
@@ -58,6 +64,9 @@ type Config struct {
 	// MetricDeclarations is a list of rules to be used to set dimensions for exported metrics.
 	MetricDeclarations []*MetricDeclaration `mapstructure:"metric_declarations"`
 
+	// MetricDescriptor is the list of overwritten metric descriptors that are sent to the CloudWatch
+	MetricDescriptors []MetricDescriptor `mapstructure:"metric_descriptors"`
```

Comment:
Could you help updating README for this new UNIT handling option?

Reply:
Updated.